### PR TITLE
added stupid route

### PR DESCRIPTION
**Pull request by toddalbert**
Added a route /stupid
returns "stupid"

Please test and make sure my code is clean.

### CODE REVIEW

**Comment by toddalbert** [comment]
Thanks for putting up with me, Jonathan

**Comment by jonathans199** [comment]
🐷

File: functions/index.js [hunk 1]
```diff
@@ -8,5 +8,6 @@ app.use(cors())
 
 app.get('/tasks', getTasks)
 app.post('/tasks', createTask)
+app.get('/stupid', (req, res) => res.send('stupid'))
```
Comment (toddalbert) [inline functions/index.js:11]: Why did you add this route?  stupid?

**Comment by jonathans199** [comment]
Michelle says to please remove stupid comments from here 🐷
